src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int height;
+    private int weight;
+
+    public Rectangle(String color, int height, int weight) {
+        super(color);
+        this.height = height;
+        this.weight = weight;
```

Comment:
According to the checklist, it is recommended to avoid passing expressions directly in the constructor. Instead, create separate variables for 'height' and 'width' before passing them to the constructor for better code readability.